libsolidity/formal/ModelCheckerSettings.cpp

```diff
@@ -62,3 +62,28 @@ bool ModelCheckerTargets::setFromString(string const& _target)
 	targets.insert(targetStrings.at(_target));
 	return true;
 }
+
+std::optional<ModelCheckerContracts> ModelCheckerContracts::fromString(string const& _contracts)
```

Comment:
I think you can remove `std` here
```suggestion
optional<ModelCheckerContracts> ModelCheckerContracts::fromString(string const& _contracts)
```

Reply:
Here `optional` is ambiguous because I'm also using `ranges`, that's why I left `std` both here and in targets above